### ISSUE TITLE
URGENT - Selection with images is not working in latest FT 6.102

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#607]
### Expected Behaviour: 
![image](https://user-images.githubusercontent.com/29519901/82916514-bf511080-9f37-11ea-9a2c-45a879e01a68.png)

### Actual Behaviour: Messages are display regarding multimedia were uploaded Ok...
![image](https://user-images.githubusercontent.com/29519901/82916579-d7289480-9f37-11ea-8fc3-5a0c66c05697.png)
But they aren´t displayed
![image](https://user-images.githubusercontent.com/29519901/82916558-cd9f2c80-9f37-11ea-8a40-77891eb74ce5.png)
### Form Name: Seleccion con Imagenes
You should reset FT application before. Try to fill out above form. with 6.90 and 6.102 versions

**Comment by nap2000** [comment]
The images are at the survey level rather than organisation level.  Other forms where the images are at the org level work.   I will get a fix out for this asap.

**Comment by atl2017** [comment]
I do not know if this is related with this new:
Organisation: DEMO
Form: Likert
When I tried to update form and replace twitter logo I got a 500 error:
![Captura21](https://user-images.githubusercontent.com/29519901/83552754-e4adc380-a4cf-11ea-9a9d-ff4320953ed8.JPG)
By the way, because likert works from collect 1.25 [https://docs.getodk.org/form-question-types/#likert-widget](https://docs.getodk.org/form-question-types/#likert-widget) we should change warning message in editor:
![Captura20](https://user-images.githubusercontent.com/29519901/83553023-3bb39880-a4d0-11ea-9300-27031b9bed37.JPG)

**Comment by nap2000** [comment]
The problem with the showing of images was caused by another form called "iconos".  This caused an error to be thrown that was badly handled by the the system that fieldTask has been built from.   This issue will be fixed in the next release.

The problem with iconos was that there were no questions in the form.

I was able to update the twitter icon.

Warning on using likert in fieldTask has been removed.
